darwin/exporter/formats/instance_mask.py

```diff
@@ -0,0 +1,99 @@
+import shutil
+import uuid
+from pathlib import Path
+from typing import Generator, List, Optional
+
+import numpy as np
+from PIL import Image
+from tqdm import tqdm
+from upolygon import draw_polygon
+
+import darwin.datatypes as dt
+
+
+def generate_instance_id(instance_ids, length=8):
+    instance_id = uuid.uuid4().hex[:length]
+    while instance_id in instance_ids:
+        instance_id = uuid.uuid4().hex[:length]
+    return instance_id
+
+
+def export(annotation_files: Generator[dt.AnnotationFile, None, None], output_dir: Path):
+    masks_dir = output_dir / "masks"
+    if masks_dir.exists():
+        shutil.rmtree(masks_dir)
+    masks_dir.mkdir(parents=True)
+    instance_ids = set()
+    with open(output_dir / "instance_mask_annotations.csv", "w") as f:
+        f.write("image_id,mask_id,class_name\n")
+        pbar = tqdm(list(annotation_files))
+        pbar.set_description(desc="Processing annotations", refresh=True)
+        for annotation_file in pbar:
```

Comment:
I like this! Perhaps we could wrap this logic around a util function, so we can reuse it elsewhere

Reply:
Good idea! Done!